src/aws/variables.tf

```diff
@@ -7,6 +7,14 @@ variable "cloudfront_price_class" {
   }
 }
 
+variable "efs_performace_mode" {
```

Comment:
```suggestion
variable "efs_performance_mode" {
```

Reply:
More importantly, is this variable used / still necessary?

Reply:
Nop indeed, it's unused